test/groovy/util/FixtureHelper.groovy

```diff
@@ -467,14 +468,14 @@ class FixtureHelper {
         result.repositories.each { repo ->
             repo.data?.git = [
                 branch                 : "origin/master",
-                commit                 : UUID.randomUUID().toString().replaceAll("-", ""),
-                previousCommit         : UUID.randomUUID().toString().replaceAll("-", ""),
-                previousSucessfulCommit: UUID.randomUUID().toString().replaceAll("-", ""),
-                url                    : "https://cd_user@somescm.com/scm/someproject/${repo.id}.git"
+                commit                 : "3a2ea1a79a134ebcb871fa8b8d6d91cf",
+                previousCommit         : "2ffb2df6aeb349ba81f5597bc0d3a087",
+                previousSucessfulCommit: "a7ff3fdbd79f456d985ff0c0fa9f4754",
+                url                    : "https://cd_user@somescm.com/scm/someproject/" + repo.id + ".git"
             ]
             repo.metadata = [
-                name       : "Sock Shop: ${repo.id}",
-                description: "Some description for ${repo.id}",
+                name       : "Sock Shop: " + repo.id,
```

Comment:
@georgfedermann why did you see the need to change how we concat Strings? I would prefer to not introduce unneeded change (and keep the established style).